selfdrive/test/test_routes.py

```diff
@@ -371,6 +371,11 @@
     'enableCamera': True,
     'enableDsu': False,
   },
+  "f33ec5db3dc94da9|2021-05-20--09-09-39": {
```

Comment:
this route doesn't have the full logs uploaded. connect your device to wifi and enable the "Upload Raw Logs" toggle until the route fully uploads

Reply:
Changed the test route to the latest on comma and awaiting upload to complete